tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -69,3 +72,88 @@ def test_conversion_as_lone_interface(self):
                 # self.check_get_timestamps()
                 # self.check_align_starting_time_internal()
                 # self.check_align_starting_time_external()
+
+
+class TestVideoInterface(DataInterfaceTestMixin, unittest.TestCase):
+    data_interface_cls = VideoInterface
+    interface_kwargs = [
+        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_avi.avi")]),
+        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_flv.flv")]),
+        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mov.mov")]),
+        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_mp4.mp4")]),
+        dict(file_paths=[str(BEHAVIOR_DATA_PATH / "videos" / "CFR" / "video_wmv.wmv")]),
+    ]
+    save_directory = OUTPUT_PATH
+
+    def check_read_nwb(self, nwbfile_path: str):
+        with NWBHDF5IO(path=nwbfile_path, mode="r", load_namespaces=True) as io:
+            nwbfile = io.read()
+            video_type = Path(self.test_kwargs["file_paths"][0]).suffix[1:]
+            assert f"Video: video_{video_type}" in nwbfile.acquisition
+
+    def check_align_starting_time_internal(self):
+        pass  # disabled since this interface follows a different strategy
+
+    def check_align_global_starting_time(self):
```

Comment:
Same note as [here](https://github.com/catalystneuro/neuroconv/pull/402/files#r1186466052)

Reply:
Adjusted the name for consistency in the Video PR